index.bs

```diff
@@ -129,25 +134,65 @@ The {{Accelerometer/z!!attribute}} attribute of the {{Accelerometer}}
 interface represents the <a>acceleration</a> along Z-axis.
 This attribute returns [=map/value=] for [=latest reading=]["z"] [=map/entry=].
 
-### Accelerometer.includesGravity ### {#accelerometer-includesGravity}
+The LinearAccelerometer Interface {#linear-accelerometer-interface}
```

Comment:
but does it really make sense? it is a fusion sensor and linear accelerometer makes it feel like one thing. Also linear acceleration sensor seems like a more common term